include/bbp/sonata/config.h

```diff
@@ -25,61 +26,131 @@
 namespace bbp {
 namespace sonata {
 
+struct SONATA_API PopulationProperties
+{
+    std::string type;
+    std::string biophysicalNeuronModelsDir;
+    std::string morphologiesDir;
+    std::map<std::string, std::string> alternateMorphologyFormats;
+};
 
-/** Read access to a SONATA circuit config file.
+/**
+ *  Read access to a SONATA circuit config file.
  */
 class SONATA_API CircuitConfig
 {
   public:
-    /** Load SONATA circuit config JSON
+
+    /**
+     * @brief Parses a SONATA JSON Config file and checks it complies with the specification.
```

Comment:
nit: `complies with the specification` not sure how much we want to guarantee we'll do this, so I'd drop that wording,

Reply:
Agree, better be safe. Removed